iroha-cli/main.cpp

```diff
@@ -36,33 +36,36 @@
 #include "responses.pb.h"
 #include "validators.hpp"
 
-DEFINE_string(config, "", "Trusted peer's ip addresses");
-// DEFINE_validator(config, &iroha_cli::validate_config);
-
-// DEFINE_string(genesis_block, "", "Genesis block for sending network");
-// DEFINE_validator(genesis_block, &iroha_cli::validate_genesis_block);
-
-DEFINE_bool(new_account, false, "Choose if account does not exist");
-DEFINE_string(name, "", "Name of the account");
-DEFINE_string(pass_phrase, "", "Name of the account");
-DEFINE_string(key_path, ".", "Path to user keys");
-
-// Sending transaction to Iroha
-DEFINE_bool(grpc, false, "Send sample transaction to IrohaNetwork");
-DEFINE_string(address, "0.0.0.0", "Address of the Iroha node");
-DEFINE_int32(torii_port, 50051, "Port of iroha's Torii");
-// DEFINE_validator(torii_port, &iroha_cli::validate_port);
+// Account information
+DEFINE_bool(
+    new_account,
+    false,
+    "Create new account, generate and save locally public/private keys.");
+DEFINE_string(account_name,
+              "",
+              "Name of the account. Must be unique in iroha network");
+DEFINE_string(pass_phrase, "", "Account pass_phrase");
```

Comment:
What is the reason of writing `pass_phrase` in CLI description?

Reply:
Typo, thanks